app/controllers/services_controller.rb

```diff
@@ -221,8 +233,7 @@ def find_services_eager_load_resources
       .includes(
         resource: [
           :addresses, :phones, :categories, :notes,
-          schedule: :schedule_days,
-          ratings: [:review]
```

Comment:
It seems on the current staging DB including ratings results in an error, even with a query such as `/services?category_id=1,2,3` that should have worked before this PR. I think we dropped this table last year, anyway.